server/app.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import debug from 'debug';
+import logger from 'morgan';
+import { config } from 'dotenv';
+import bodyParser from 'body-parser';
+import apiRoutes from './routes';
+
+const debugged = debug('app');
+config();
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+app.use('/', apiRoutes);
+
```

Comment:
considering that there was an error handler in the original file,

```javascript
/// catch 404 and forward to error handler	
app.use((req, res, next) => {	
    const err = new Error("Not Found");	
    err.status = 404;	
    next(err);	
});	

 /// error handlers	

 // development error handler	
// will print stacktrace	
if (!isProduction) {	
    app.use((err, req, res, next) => {	
        console.log(err.stack);	

         res.status(err.status || 500);	

         res.json({	
            errors: {	
                message: err.message,	
                error: err	
            }	
        });	
    });	
}	

 // production error handler	
// no stacktraces leaked to user	
app.use((err, req, res, next) => {	
    res.status(err.status || 500);	
    res.json({	
        errors: {	
            message: err.message,	
            error: {}	
        }	
    });	
});
```

try improving on it or just put it back